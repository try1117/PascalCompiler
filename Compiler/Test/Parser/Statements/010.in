program test;
var a: integer;
begin
  a;
end.
