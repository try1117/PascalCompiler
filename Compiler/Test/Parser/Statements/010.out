Syntax exception in position (4,4) - Expected ":=" but found ";"
